### DISCUSSION TITLE
Interactive Playground

### DISCUSSION THREAD

**Discussion by ulrikandersen** [cjbooms/fabrikt#344] [Ideas]
What if we gave people the opportunity to try Fabrikt with minimal investment? 

This would allow them to see, firsthand, the high-quality Kotlin code that can be generated from their OpenAPI specifications. By showcasing the results, we hope to motivate users to invest the time and effort needed to integrate Fabrikt into their projects.

This idea sparked the creation of an initial version of an interactive playground 🛝

Using a blend of Ktor, Kotlin HTML DSL, a dash of HTMX, and some excellent libraries for YAML editing and Kotlin syntax highlighting, an early version is now available at [try.fabrikt.ulrikandersen.dk](https://try.fabrikt.ulrikandersen.dk) (try.fabrikt.io in the future?) ([source code here](https://github.com/ulrikandersen/fabrikt/tree/playground/playground)).

👂 I’d love to hear your thoughts on this concept, feedback on the current version, and your opinion on whether this is something worth refining and sharing more broadly with the community.

**Comment by ulrikandersen** [comment]
Moved to #348

**Comment by cjbooms** [comment]
Very sorry I missed this. Great idea

**Reply by ulrikandersen** [reply]
No worries, figured I could just as well open a PR since I already wrote the code 🙂
